src/sentry/scim/endpoints/members.py

```diff
@@ -604,6 +619,13 @@ def post(self, request: Request, organization) -> Response:
                     referrer=request.data.get("referrer"),
                 )
 
+            metrics.incr(
+                "sentry.scim.member.provision",
+                tags={"organization": organization},
+            )
+            if update_role:
+                metrics.incr("sentry.scim.member.update_role", tags={"organization": organization})
+
```

Comment:
Check that the role was explicitly set to see if we need to send the `update_role` metric when provisioning a new member